src/pages/experiments/[experimentId]/data-processing/index.jsx

```diff
@@ -324,7 +324,7 @@ const DataProcessingPage = ({ experimentId, experimentData, route }) => {
                             disabledByPipeline
                           }
                         >
-                          {processingConfig[key]?.enabled === false ? (
+                          {!processingConfig[key]?.enabled || processingConfig[key]?.preFiltered ? (
```

Comment:
The ticked describes this as a per-sample setting: will there be a experiment-wide `preFiltered`?

Reply:
Changed this in 72629e1 to per-experiment wide disable detecting the key `preFiltered` in sample.